openpmd_viewer/openpmd_timeseries/main.py

```diff
@@ -630,7 +630,9 @@ def _find_output(self, t, iteration):
                       "The requested iteration '%s' is not available.\nThe "
                       "available iterations are: \n - %s\n" % (iteration, iter_list))
         else:
-            pass  # self._current_i retains its previous value
+            raise OpenPMDException(
+                "Please pass either a time (`t`) \nor an "
+                "iteration (`iteration`).")
```

Comment:
I would probably print this in a single line:
```suggestion
                "Please pass either a time (`t`) or an "
                "iteration (`iteration`).")
```